app/models/scc_account.rb

```diff
@@ -287,9 +287,13 @@ def invalidate_subscription_status(items_to_invalidate, save_record:)
   end
 
   def self.download_policy_selection_values
-    names = { 'on_demand' => _('On Demand'), 'immediate' => _('Immediate') }
+    names = {
+      ::Katello::RootRepository::DOWNLOAD_ON_DEMAND => _('On Demand'),
+      ::Katello::RootRepository::DOWNLOAD_IMMEDIATE => _('Immediate')
```

Comment:
Shouldn't it be like this: 
```suggestion
      'on_demand' => ::Katello::RootRepository::DOWNLOAD_ON_DEMAND,
      'immediate' => ::Katello::RootRepository::DOWNLOAD_IMMEDIATE
```